testapp/test_bash_evaluation.py

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Move all the tests into a separate "tests" directory in the appropriate place with an `__init__.py` also in the directory.